### DISCUSSION TITLE
Video Camera

### DISCUSSION THREAD

**Discussion by indium114** [mortuusars/Exposure#108] [Ideas]
Add a new kind of film, Video Film (B&W, Color) which when put into a camera will record video! The B&W film can hold 1min of video, and the Color film can hold 3mins!

They will both be developed with dragon’s breath, or something else hard to get.

They can be spliced together and wound onto reels with a new workstation block.

The reel can be put into a Projector and projected onto a screen made of white concrete!

Filming can be done by someone holding the camera, or with the new Tripod block.

When on the tripod, redstone can be ran from it to an activation device to start/stop recordings!

I know this is quite complex and likely won’t be added, but I thought I would share my idea!

-StikyPiston :D

**Comment by unbreakify** [comment]
Would really love to see this too. The issue i can see is potetionally being too storage-heavy, but i dont really see it being too much of an issue in the modern-PC era (if done with PNGS), but framerate might be a problem. Potetionally something like Super 8 could be added, which is 18 FPS, and could be a viable contender for video-film, as it could help with such memory problems. If Super 8 film is used, and the math is done -  

777kb (for a gradient PNG 256x256) * 60 seconds = 26.76 MB for an entire minute of film if done with super 8. an entire stack, 46620 (KB of 60 second film) * 64 = 2.98 gigabytes. **Not good**. 

ome sort of compression would have to be done for this to be feasible. Yeah, yapfest i know, but its 100% something that would have to be taken into consideration if this was added.

**Reply by indium114** [reply]
It could also perhaps have some way to change the path where videos are stored, outside of the game directory, so you could put video clips onto a large SSD. It could also maybe use a smaller format like WebM, or even a GIF if you're willing to have shorter videos.

**Reply by unbreakify** [reply]
GIF's might actually be a good solution to this problem! Film could potetionally be lower quality than regular photos to account for the file size too.
